docs/book/src/blockchain-development/native_assets.md

```diff
@@ -1,32 +1,457 @@
-# Native Support for Multiple Asset Types
+# Native Assets
 
 <!-- This section should explain native assets in Sway -->
 <!-- native_assets:example:start -->
 The FuelVM has built-in support for working with multiple assets.
 
```

Comment:
A bullet list of links to each section would be really helpful at the top here to easily jump down to a section.

Reply:
This is currently forbidden by CI

Reply:
@bitzoic what CI test forbids this?

Reply:
Yes I had the same issue in https://github.com/FuelLabs/sway/pull/5715